src/Sylius/Bundle/ProductBundle/Doctrine/ORM/ProductAttributeValueRepository.php

```diff
@@ -79,4 +91,12 @@ public function createByProductCodeAndLocaleQueryBuilder(
             ->setParameter('locale', $localeCode)
         ;
     }
+
+    protected function isPostgres(Connection $connection): bool
```

Comment:
Better to match naming.

```suggestion
    protected function isPostgreSQLPlatform(Connection $connection): bool
    protected function isPostgreSQLDatabasePlatform(Connection $connection): bool
```